packages/vuetify/src/mixins/activatable/index.ts

```diff
@@ -115,6 +123,16 @@ export default baseMixins.extend({
         }
       }
 
+      if (this.openOnFocus) {
+        listeners.focus = (e: FocusEvent) => {
+          this.getActivator(e)
+
+          e.stopPropagation()
```

Comment:
transpose this with line 128

Reply:
Not sure what you mean...?

Reply:
Ah. Put `e.stopPropagation()` before `this.getActivator(e)`?